CHANGELOG.adoc

```diff
@@ -37,6 +37,7 @@
 === Improvements
 
 - https://github.com/eclipse-sirius/sirius-web/issues/5608[#5608] [diagram] Hide empty sections on the palette 
+- https://github.com/eclipse-sirius/sirius-web/issues/5416[#5416] [diagram] Move PaletteToolList component to diagram package
```

Comment:
How is that moving PaletteToolList TO diagram package when it's 100% the other one around. On top of that PaletteToolList is now in `sirius-components-palette` with diagram specific props like `diagramElementId`...